contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -3135,6 +3135,12 @@ void extractQueryHintsFromOptionClause(TSqlParser::Option_clauseContext *octx)
 		if (option->TABLE())
 		{
 			std::string table_name = ::getFullText(option->table_name());
+			size_t idx = table_name.find('.');
+			while (idx != std::string::npos)
+			{
+				table_name = table_name.substr(idx + 1);
+				idx = table_name.find('.');
+			}
```

Comment:
please use `getFullText(option->table_name()->table())` to obtain a table name instead of manually finding `.`. table can be delimited like "schema.name.having.dot"."table.name.having.dot" so we should rely on the parser to split an identifier into schema name and table name. It would be good to have relevant tests too.